src/test/java/uk/nhs/digital/nhsconnect/nhais/repository/SequenceRepositoryTest.java

```diff
@@ -0,0 +1,51 @@
+package uk.nhs.digital.nhsconnect.nhais.repository;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.springframework.data.mongodb.core.FindAndModifyOptions;
+import org.springframework.data.mongodb.core.MongoOperations;
+import org.springframework.data.mongodb.core.query.Query;
+import org.springframework.data.mongodb.core.query.Update;
+import uk.nhs.digital.nhsconnect.nhais.model.sequence.SequenceId;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.eq;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.Silent.class)
```

Comment:
I've been doing tests with JUnit5 - `@ExtendWith(MockitoExtension.class)`

Reply:
I've also been using `@InjectMocks` and `@Autowiring` dependencies instead of using constructor injection

Reply:
I'm updating both test class to use:
org.junit.jupiter.api.Test